packages/web-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -2,38 +2,41 @@ import React, { forwardRef, ForwardedRef } from 'react';
 import classNames from 'classnames';
 import { useStyleProps } from '../../hooks';
 import { SpiritCheckboxProps } from '../../types';
-import { useValidationText } from '../Field';
+import { HelperText, ValidationText } from '../Field';
+import useAriaDescribedBy from '../Field/useAriaDescribedBy';
```

Comment:
this could be exported from '../Field' too, couldn't it?

Reply:
this applies to all the components

Reply:
imports fixed